coretex/cli/main.py

```diff
@@ -61,7 +61,7 @@ def update() -> None:
 
 
 @click.group(cls = ClickExceptionInterceptor)
-@utils.onBeforeCommandExecute(utils.checkLibVersion)
+@utils.onBeforeCommandExecute(utils.checkLibVersion, excludeOptions = ["update"])
```

Comment:
Will this exclude `coretex node update` as well, or just `coretex update` command?

Reply:
just coretex update command.

Reply:
This is still using `excludeOptions`, shouldn't it use `excludeSubcommands`?